.github/workflows/dotnet-build-and-test.yml

```diff
@@ -98,9 +98,9 @@ jobs:
           AzureOpenAI__DeploymentName: ${{ vars.AZUREOPENAI__DEPLOYMENTNAME }}
           AzureOpenAIEmbeddings__DeploymentName: ${{ vars.AZUREOPENAIEMBEDDING__DEPLOYMENTNAME }}
           AzureOpenAI__Endpoint: ${{ secrets.AZUREOPENAI__ENDPOINT }}
-          AzureOpenAIEmbeddings__Endpoint: ${{ secrets.AZUREOPENAI__ENDPOINT }}
+          AzureOpenAIEmbeddings__Endpoint: ${{ secrets.AZUREOPENAI_EASTUS__ENDPOINT }}
```

Comment:
What is the source of this configuration? Because I can't find it in secrets/variables in GitHub settings. The reason why I switched this setting is because `text-embedding-3` models are supported in limited regions of Azure OpenAI resource, and `eastus` is one of them.